spikeinterface/sorters/external/kilosort2_5.py

```diff
@@ -198,7 +202,7 @@ def _get_specific_options(cls, ops, params):
         ops['NT'] = params['NT']  # must be multiple of 32 + ntbuff. This is the batch size (try decreasing if out of memory).
         ops['whiteningRange'] = 32.0  # number of channels to use for whitening each channel
         ops['nSkipCov'] = 25.0  # compute whitening matrix from every N-th batch
-        ops['scaleproc'] = 200.0  # int16 scaling of whitened data
+        # ops['scaleproc'] = 200.0  # int16 scaling of whitened data
```

Comment:
Why not keeping it as default value?

Reply:
Because it is done later.
When skip the preprocessing the user must give it explicitly.
See the logic L216-L222